File: react-app/src/app/models/stafi.ts
import { Qyteti } from './qyteti';
import {Shteti} from './shteti';
import {Gjinia} from './gjinia';
import {Banka} from './banka';

export interface Stafi{
    stafiId?: number;
    emri: string;
    mbiemri: string;
    detyra: string;
    shtetiId: string;
    shteti?: Shteti;
    qytetiId: number;
    qyteti?: Qyteti;
    gjiniaId: number;
    gjinia?: Gjinia;
    bankaId: number;
    banka?: Banka;
    adresa: string;
    dataLindjes: string;
}

File: Domain/Stafi.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    public class Stafi
    {
        public int StafiId { get; set; }
        public string Emri { get; set; } = string.Empty;
        public string Mbiemri { get; set; } = string.Empty;
        public string Detyra { get; set; } = string.Empty;
        public int ShtetiId { get; set; }
        public Shteti? Shteti { get; set; }
        public int QytetiId { get; set; }
        public Qyteti? Qyteti { get; set; }
        public int GjiniaId { get; set; }
        public Gjinia? Gjinia { get; set; }
        public int BankaId { get; set; }
        public Banka? Banka { get; set; }
        public string Adresa { get; set; } = string.Empty;
        public DateTime DataLindjes { get; set; }
    }
}
